starter_code/app.js

```diff
@@ -1,18 +1,31 @@
 
 const express = require('express');
-const hbs     = require('hbs');
-const app     = express();
-const path    = require('path');
+const hbs = require('hbs');
+const app = express();
+const path = require('path');
 const PunkAPIWrapper = require('punkapi-javascript-wrapper');
 const punkAPI = new PunkAPIWrapper();
 
 app.set('view engine', 'hbs');
-app.set('views', __dirname + '/views');
+app.set('views', path.join(__dirname, '/views'));
+hbs.registerPartials(path.join(__dirname, '/views/partials'));
 app.use(express.static(path.join(__dirname, 'public')));
 
 app.get('/', (req, res, next) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res, next) => {
+  punkAPI.getBeers()
+    .then(beers => {
+      const beerArray = [];
+      beers.forEach(beer => beerArray.push(beer.name));
```

Comment:
You didn't had to do this here. In the partial you could also choose which data to use.